addon/components/bs-modal/dialog.hbs

```diff
@@ -13,9 +13,8 @@
   {{did-insert this.getOrSetTitleId}}
   {{did-insert this.setInitialFocus}}
 >
-  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}}" role="document">
-    <div class="modal-content"
-     tabindex="-1"
+<div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}} {{if @fullscreen (if (bs-eq @fullscreen true) "modal-fullscreen" "modal-fullscreen-{{@fullscreen}}-down")}}" role="document">
```

Comment:
I just noticed that fullscreen modals are only supported by Bootstrap 5. Ember Bootstrap supports both Bootstrap 4 and 5. We need to exclude this code if Ember Bootstrap is used with Bootstrap 4.

We are using build-time macros provided by Embroider to do so: `{{macroCondition}}` to have dead-code elimination at build-time. `{{macroGetOwnConfig}}` to get the value to decide, which code path should be eliminated. Together it looks like this:

```hbs
{{macroCondition (macroGetOwnConfig "isBS5") "only-included-in-the-build-if-app-is-using-bootstrap-5"}}
```

You can see an example here: https://github.com/kaliber5/ember-bootstrap/blob/4b261eba2d6352eafdade589e4ba2a5a9f3e750a/addon/components/bs-button.hbs#L4

Please also update the API docs to mention that it is only supported for Bootstrap 5.

Reply:
Well done !
Fixed in doc and in hbs